nvme-print.c

```diff
@@ -1773,17 +1773,23 @@ static void nvme_show_id_ctrl_cmic(__u8 cmic)
 static void nvme_show_id_ctrl_oaes(__le32 ctrl_oaes)
```

Comment:
Humen -> Human

"Add human readable for the Zoned Namespace Command Set specific Identify Namespace data structure"

Reply:
@MatiasBjorling 
	if (human)
		show_nvme_id_ns_zoned_zoc(ns->zoc);
	printf("ozcs:   %u\n", le16_to_cpu(ns->ozcs));
	if (human)
		show_nvme_id_ns_zoned_ozcs(ns->ozcs);

already added